test/e2e/framework/pod/create.go

```diff
@@ -171,22 +196,22 @@ func MakeSecPod(ns string, pvclaims []*v1.PersistentVolumeClaim, inlineVolumeSou
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      podName,
-			Namespace: ns,
+			Namespace: podInfo.ns,
```

Comment:
I think namespace should be required, can we add a check that in the beginning that it's non-empty?

Reply:
Should I return an error if it is empty or default it to some value?

Reply:
I think returning an error would be appropriate. This would indicate that the test is not written properly if it doesn't pass in a namespace.